src/libraries/System.Net.Http/src/System/Net/Http/HttpRequestOptions.cs

```diff
@@ -0,0 +1,117 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+
+namespace System.Net.Http
+{
+    public sealed class HttpRequestOptions : IDictionary<string, object?>
+    {
+        private IDictionary<string, object?> Options { get; } = new Dictionary<string, object?>();
```

Comment:
This can be typed as Dictionary rather than IDictionary.

Reply:
Addressed in recent commits